layouts/docs/baseof.html

```diff
@@ -24,6 +24,11 @@
             {{ if not .Site.Params.ui.breadcrumb_disable }}{{ partial "breadcrumb.html" . }}{{ end }}
             {{ block "deprecated" . }}
               {{ partial "deprecation-warning.html" . }}
+            {{ end }}
+            {{ if .HasShortcode "kat-button" }}
```

Comment:
Not sure how to make the condition more generic. For example, page 'https://deploy-preview-34005--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/explore/explore-interactive/' doesn't have such a warning.

A more straightforward approach would be add this warning to those pages directly, and then remove the whole page completely when the day comes.

Reply:
I've added a new shortcode to those pages. We can now detect all Katacoda use for the English localization, and updating other localizations is only a search-and-replace away.